seed/tests/test_geocode_views.py

```diff
@@ -64,70 +69,90 @@ def test_geocode_confidence_summary_returns_summary_dictionary(self):
         property_manual = PropertyState(**property_manual_details)
         property_manual.save()
 
+        self.property_view_factory.get_property_view(state=property_manual)
+
         property_missing_details = self.property_state_factory.get_details()
         property_missing_details["organization_id"] = self.org.id
         property_missing_details["geocoding_confidence"] = "Missing address components (N/A)"
         property_missing = PropertyState(**property_missing_details)
         property_missing.save()
 
+        self.property_view_factory.get_property_view(state=property_missing)
+
         tax_lot_none_details = self.tax_lot_state_factory.get_details()
         tax_lot_none_details["organization_id"] = self.org.id
         tax_lot_none = TaxLotState(**tax_lot_none_details)
         tax_lot_none.save()
 
+        self.taxlot_view_factory.get_taxlot_view(state=tax_lot_none)
+
         tax_lot_high_details = self.tax_lot_state_factory.get_details()
         tax_lot_high_details["organization_id"] = self.org.id
         tax_lot_high_details["geocoding_confidence"] = "High (P1AAA)"
         tax_lot_high = TaxLotState(**tax_lot_high_details)
         tax_lot_high.save()
 
+        self.taxlot_view_factory.get_taxlot_view(state=tax_lot_high)
+
         tax_lot_low_details = self.tax_lot_state_factory.get_details()
         tax_lot_low_details["organization_id"] = self.org.id
         tax_lot_low_details["geocoding_confidence"] = "Low (P1CCC)"
         tax_lot_low = TaxLotState(**tax_lot_low_details)
         tax_lot_low.save()
 
+        self.taxlot_view_factory.get_taxlot_view(state=tax_lot_low)
+
         tax_lot_manual_details = self.tax_lot_state_factory.get_details()
         tax_lot_manual_details["organization_id"] = self.org.id
         tax_lot_manual_details["geocoding_confidence"] = "Manually geocoded (N/A)"
         tax_lot_manual = TaxLotState(**tax_lot_manual_details)
         tax_lot_manual.save()
 
+        self.taxlot_view_factory.get_taxlot_view(state=tax_lot_manual)
+
         tax_lot_missing_details = self.tax_lot_state_factory.get_details()
         tax_lot_missing_details["organization_id"] = self.org.id
         tax_lot_missing_details["geocoding_confidence"] = "Missing address components (N/A)"
         tax_lot_missing = TaxLotState(**tax_lot_missing_details)
         tax_lot_missing.save()
 
-        url = reverse('api:v2:geocode-confidence-summary')
+        self.taxlot_view_factory.get_taxlot_view(state=tax_lot_missing)
+
+        url = reverse('api:v3:geocode-confidence-summary')
         post_params = {
             'organization_id': self.org.pk,
-            'property_ids': [
+            'property_view_ids': [
                 property_none.id,
                 property_high.id,
                 property_low.id,
                 property_manual.id,
                 property_missing.id
             ],
-            'tax_lot_ids': [
+            'taxlot_view_ids': [
```

Comment:
Similar to the other comment, I think we want the -View IDs for the -View objects created in the new lines above.